Logos/Perimeter81_Logo.svg

```diff
@@ -0,0 +1 @@
A2klEQVQoU6XTMUqDQRDF8d9GESRfLmDAaBtvYCGYzlYPkEuYU+QUHkCPkMLG3sLGxgTSWyxYrkXmw41KiOY1A8P7D8vM21RKUas/miQ0OI7WAnk5m64ZUwtWwABDXIfnAS+Y1wNSKUV/NGlwEsBN1EGA8wDvo74tZ9Ocji5vG1xgXAENUoAFuRpwh8f9MI5x9Q1oldDDmeoVHV+L+A2qVS+t6VTNbZWgBf+sncH1FGxWYQVmkQ6bB7RnWSDv9U7PP/CObhgOceDnHV/xZHXH5/8nZ6es1tr2d3wCZ8paY7TywP0AAAAASUVORK5CYII="/><polygon class="cls-6" points="42.7 24.74 37.52 27.71 37.5 27.7 32.29 24.73 32.29 24.69 37.49 21.7 42.7 24.74"/><polygon class="cls-7" points="42.7 24.74 42.7 30.69 37.5 33.64 37.5 27.7 37.52 27.71 42.7 24.74"/><polygon class="cls-8" points="37.5 27.7 37.5 33.64 37.49 33.64 32.29 30.69 32.29 24.73 37.5 27.7"/><line class="cls-9" x1="33.29" y1="25.34" x2="33.29" y2="25.34"/><g class="cls-10"><g class="cls-11"><image width="91" height="79" transform="translate(-8.11 -18.7)" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAFsAAABPCAYAAACEcbkUAAAACXBIWXMAAAsSAAALEgHS3X78AAAF9klEQVR4Xu3az27bSBLH8V9VdZN0lEixgcEgQQabQw4L++gXyGFeIfs6SZ5nXiK3Pflon+YwAYI1gsms138oieyuqj0ojh3PYgwMNk1ht78nQSIp6gOqxSZF7o5amfi+BWr/vSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wSp2wcJ9C0wdEdHm0fUfQN/SzauvPz+5ecq3/F+itK37t0F2bHCPCTggvARw+eQG+9Gp4x0AHDtw4Bt82lr0rcP+GnmfcHjGB+td6vue85MzsnH2BZub3sPprodw6T8vWsPRqQOwzatvfNvQtwp7A/3mC/KL84GXi1bGq3/J/OFjzqOyJmPMAVwAJKOHSNau5nr5MGvzW2Oz2Xs7PoYBMOC1u2NrPuDWYBOBgLeEl+AXH/ZkufinPLr6QVbNECJRzJElJhaVQa7X4UQ2RtNGQx405m5Y5ssHgz5atnlzpO8a8Mq2ZWjZCuwN9E+MwzN+/tup9LPvwiyOQaxrhBAHz40QRWYLrpHdMxEFJyFTNY2OpO7J0IzJKc0ypb4/z4vFhR4f7ytw4tswrEyOfRv62cc+zGaLcEWrJlBsqLWWzFtybQjSKFkIEthdCQDIyEw0OzyFJCMRDWAajNNo58txvd5Nu7uneQP+yqYeUibF3kD/jXH4Iz/72IeuC5EXbYNsrSl3kNTBYwtoo24NM4kbiYgREXtWc2ZXB5KDR2hYQ3xNWdcUd9Z2nsZx3B0/fH+Wr4eUKcEnw/7yY/gS/PwXhLb9IfrDs9ZS03mQzlPYgeSOwA3cGiMJ4s7Ozu62ObKVnVjNnLILUlAMWWxNxitSWkkbVnQVhmHg9Mvz9xnvYFMOJxPOIB3APh38Cl7NZ7JqhrCytoGhReYOkjs4Wrg1TB7EP09ulJ0sGCk7ALgzMXkgRYSjhcUW5i21oVnZ2KyaIazmn+TgVzCwT5juwJ4S+y0BJ9T34MfDHjchh8hNyESRCJHVAzMJnEkNZp7UHQmkA0gHsK9BNLhQNhYFAA9MwiTMIWSMMbKFJuTweNjjYdgj4IS+noGWbRLsmyk4kPOcUsqsaYc1jRJCZDdnAFDNJvAEooGMVgztnXDlhCuD9gRfkvvSFYM4kpkr1ByuFJ3J1FiTcUqZc3705T1vv3/Jpr02cviE8LEHALiPRBw8JXUKUAZleNRM5myuCs4kki0nIxJ3J0JgtmRBhEQlCFzJDWaRMzKMJDqzuAPI+Yxw+IRwdPrH+/QNmxb76Ah49lcwD04SDdnUNWeORJZJCQYOUY1yVkKmccg5NErr7N4FkqRs3AQiDZI0uDgjiCNvzr1NTTlGMx48hF3H0d8BPL1vr75Z02LjqXddsLYVO9eYAwZmBo05aOOZRgouyJoguSWktYrG2Cmn0W1Y0jB7xC1SyMJCGUFJBO4gMYdJdo9pPaquWrEFLh14Ot2vIybC9s0gAOAnb9sTPz+fa9dpXrcP0Aobq3EyJZEdE6xMjTOtsuYHg9Kn0ULY9Zx7sk4pXDWifCHrHGUntqzpgplbR2RLo+pyvMxdH/XnxYUD+w68cvfXk6BPgr2JALzx42PYixeB1uucFu1gq6ugmK+JAJCYt2vRZSfWrLJ+jHPDh1MHLjab+H5GL7TX/Mk5781ZlomhTEACycyH9sK6PtpicaEfjmGbaTthqtO/ySY1wK2LTwAfHID7HpzznMwuiP8y/4PLp9dH5u3LsKc0DHt0c9bxHiHs3Vl32quAk2IDt8H3P58H/4NweAjgCDh6+vmrf+L/6cbA728wvAJwsrnJ8O56qZv1p4QGtgAbuIt2t/vvvnx96+zuNu5fv1RbgX3d3cnGnwW63s6fXf9btVXY/+tNMl3/f61iF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF6xiF+zfNXFuX4C0038AAAAASUVORK5CYII="/></g></g></g></g></svg>
```

Comment:
Please update the svg logo to not have cls style formats. Pasting the following on logo requirements from the data connector guidance: 
The company logo adhering to the following requirements in the 'Logo' folder 
Logo needs to be in SVG format
Ensure raw file of logo doesn’t have cls style formats
Logo scales well to fit in a 75 px square